### PR TITLE
Translate contribute/new-content/overview into Japanese

### LINKED ISSUE #27496: Translate contribute/new-content/overview/ into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate [contribute/new-content/overview/](https://kubernetes.io/docs/contribute/new-content/overview/) into Japanese.

**Why is this needed**

This page has not been translated.

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by shuuji3**
✅  Resolves #27496.

Page preview:
- https://deploy-preview-27499--kubernetes-io-master-staging.netlify.app/ja/docs/contribute/new-content/
- https://deploy-preview-27499--kubernetes-io-master-staging.netlify.app/ja/docs/contribute/new-content/overview/

### CODE REVIEW

File: content/ja/docs/contribute/new-content/overview.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+このセクションでは、新しいコンテンツの貢献を行う前に知っておくべき情報を説明します。
+
+<!-- body -->
+
+## 貢献の基本
+
+- KubernetesのドキュメントはMarkdownで書き、Kubernetesのウェブサイトは[Hugo](https://gohugo.io/)を使ってビルドします。
+- ソースは[GitHub](https://github.com/kubernetes/website)にあります。Kubernetesのドキュメントは`/content/en/docs/`にあります。リファレンスドキュメントの一部は、`update-imported-docs/`ディレクトリ内のスクリプトから自動的に生成されます。
+- [Page content types](/docs/contribute/style/page-content-types/)はHugo内のドキュメントのコンテンツの見え方を？指定します？
```
Comment (atoato88) [inline content/ja/docs/contribute/new-content/overview.md:19]: It is a difficult part for translation.. :sweat_smile: 
How about my suggest following?
```suggestion
- [Page content types](/docs/contribute/style/page-content-types/)にHugoによるドキュメントのコンテンツの見え方を記述しています。
```

**Review by atoato88 — COMMENTED** [review]
Here are some review comments.

File: content/ja/docs/contribute/new-content/overview.md (hunk 1 above)
Comment (shuuji3) [reply content/ja/docs/contribute/new-content/overview.md:19]: I'm sorry, I thought I would fix it later but forgot it... 😅

I think your translation is perfect.

File: content/ja/docs/contribute/new-content/overview.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+シナリオ | ブランチ
+:---------|:------------
+現在のリリースに対する既存または新しい英語のコンテンツ | `master`
+機能変更のリリースに対するコンテンツ | 機能変更が含まれるメジャーおよびマイナーバージョンに対応する、`dev-<version>`というパターンのブランチを使います。たとえば、機能変更が`v{{< skew nextMinorVersion >}}`に含まれる場合、ドキュメントの変更は``dev-{{< skew nextMinorVersion >}}``ブランチに追加します。
+他の言語内のコンテンツ(翻訳) | 各翻訳対象の言語のルールに従います。詳しい情報は、[翻訳のブランチ戦略](/docs/contribute/localization/#branching-strategy)を読んでください。
+
+それでも選ぶべきブランチがわからないときは、Slack上の`#sig-docs`チャンネルで質問してください。
+
+{{< note >}}
+すでにpull requestを作成していて、ベースブランチが間違っていたことに気づいた場合は、作成者であるあなただけがベースブランチを変更できます。
```
[the remark this answers is not in the document]
Comment (shuuji3) [reply content/ja/docs/contribute/new-content/overview.md:45]: Thanks for pointing it out. I didn't notice but it should be a typo.

**Comment by shuuji3** [comment]
Thanks @atoato88, I've just fixed two typos.

**Comment by atoato88** [comment]
/lgtm
Thank you for quick updating! :smile:

**Pull request merged**